cmd/nerdctl/image/image_inspect.go

```diff
@@ -95,7 +97,13 @@ func imageInspectAction(cmd *cobra.Command, args []string) error {
 	}
 	defer cancel()
 
-	return image.Inspect(ctx, client, args, options)
+	err = image.Inspect(ctx, client, args, options)
+	if uw, ok := err.(interface{ Unwrap() []error }); ok {
+		errs := uw.Unwrap()
+		err = fmt.Errorf("%d errors: %v", len(errs), errs)
+	}
```

Comment:
why do we need to unwrap here?

Reply:
Thanks @djdongjin, I just thought it would be good to align with the format used by `nerdctl inspect` command.
https://github.com/containerd/nerdctl/blob/6f34536582f9c7211883432580e732c1ac3a49f0/cmd/nerdctl/inspect/inspect.go#L164

```bash
austin@nimbus:~/workspace/nerdctl$ mysudo nerdctl inspect dne:latest dne2:latest
FATA[0000] 2 errors: [no such object dne:latest no such object dne2:latest]
```

Open to suggestions here. Docker is having format of one error per line, but thought perhaps to stay consistent with nerdctl inspect instead.

```
austin@nimbus:~/workspace/nerdctl$ docker inspect dne:latest dne2:latest
[]
Error: No such object: dne:latest
Error: No such object: dne2:latest
austin@nimbus:~/workspace/nerdctl$ docker image inspect dne:latest dne2:latest
[]
Error response from daemon: No such image: dne:latest
Error response from daemon: No such image: dne2:latest
```

Reply:
I feel one error per line will be visiually better? Also some other nerdctl command also does one error per line:

```
☁  ~   ➜ sudo nerdctl rmi sdaf sdfasdf
FATA[0000] 2 errors:
no such image: sdaf
no such image: sdfasdf
```

Reply:
SGTM, pushing new revision with suggestion.